### ISSUE TITLE
Unable to Parse schema with `@unique` directive - CLI

### ISSUE THREAD

**Issue by khaifahmi99** [grafbase/grafbase#42]
**Description**
Using the CLI, Graphql schema cannot be parsed when running `grafbase dev`. This happened whenever the `@unique` directive is included in the schema. 

```
type User @model {
  id: ID!
  email: String! @unique
  name: String!
  posts: [Post]
}

type Post @model {
  id: ID!
  slug: String! @unique
  title: String!
  content: String!
  user: User!
  comments: [Comment]
}

type Comment @model {
  id: ID!
  content: String!
  user: User!
  post: Post!
}
```

**To Reproduce**
1. `npm i -g grafbase`
2. `grafbase init test-project`
3. Replace the content of the `schema.graphql` to snippet above
4. Run `grafbase dev`

**Expected behavior**
There should be NO error when parsing and the service should start locally (endpoint or playground)

**Desktop**
- OS: macOS Intel Chip, Monterey 12.5
- Grafbase CLI Version: 0.3.0

**Comment by yoav-lavi** [comment]
Hey @khaifahmi99!

First off, thank you for the report! We really appreciate you taking the time to let us know!

Regarding the issue itself - Grafbase CLI `0.3.0` was released before the `@unique` directive was implemented, and so the next version we release will support `@unique`.

We're currently working on the upcoming version and I'll make sure to update you here personally once it's live!

**Comment by yoav-lavi** [comment]
@khaifahmi99 we just released `0.4.0` which solves this issue!
